An optional patch that allows getauxval() to be used on musl libc (for Void
Linux), bypassing the SIGILL handler based checks. Makes an assumption about
a modern Linux userland, which is usually not a problem.

--- a/crypto/ppccap.c
+++ b/crypto/ppccap.c
@@ -133,11 +133,9 @@ size_t OPENSSL_instrument_bus2(unsigned int *out, size_t cnt, size_t max)
         return 0;
 }
 
-#if defined(__GLIBC__) && defined(__GLIBC_PREREQ)
-# if __GLIBC_PREREQ(2, 16)
-#  include <sys/auxv.h>
-#  define OSSL_IMPLEMENT_GETAUXVAL
-# endif
+#ifdef __linux__
+# include <sys/auxv.h>
+# define OSSL_IMPLEMENT_GETAUXVAL
 #endif
 
 /* I wish <sys/auxv.h> was universally available */
